xarray/core/rolling.py

```diff
@@ -153,14 +153,21 @@ def _reduce_method(  # type: ignore[misc]
             array_agg_func = getattr(duck_array_ops, name)
 
         bottleneck_move_func = getattr(bottleneck, "move_" + name, None)
+        if module_available("numbagg"):
+            import numbagg
+
+            numbagg_move_func = getattr(numbagg, "move_" + name, None)
+        else:
+            numbagg_move_func = None
 
         def method(self, keep_attrs=None, **kwargs):
             keep_attrs = self._get_keep_attrs(keep_attrs)
 
             return self._numpy_or_bottleneck_reduce(
```

Comment:
```suggestion
            return self._array_reduce(
```

Reply:
Are you suggesting we should rename this method? (that method doesn't exist atm I think...)

Reply:
Yes sorry that wasn't clear. 	`_numpy_or_bottleneck_numbagg_reduce` isn't very nice.